config/core/200-serviceaccount.yaml

```diff
@@ -19,13 +19,18 @@ metadata:
   namespace: knative-serving
   labels:
     serving.knative.dev/release: devel
+    app.kubernetes.io/name: controller
```

Comment:
Can you just comment on why certain kinds get the `name` label and others don't. 

From this file I see `ServiceAccount` & `ClusterRoleBinding` get the `name` label but `ClusterRole` does not

Reply:
I didn't add the label to resources that aren't specific to a particular app.